add.js

```diff
@@ -0,0 +1,38 @@
+const filterNumbers = (numbers) => {
+  return numbers
+    .split("+")
+    .map((ele) => parseInt(ele))
+    .filter((ele) => ele <= 1000);
```

Comment:
I suggest moving this to another function and calling it removeThousands for example. This Will help you achieve the separation of concerns w talked about earlier and make your code more readable.

Reply:
Done